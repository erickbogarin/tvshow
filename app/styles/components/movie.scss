
//.movie-box { }

.movie-header {
  background-color: #262626;
  box-shadow: 0 3px 6px #191919;

  color: white;

  padding: 130px 0;
  text-align: center;

  > .title {
    border-bottom: 5px solid #47CF73;
    font-size: 1.5em;
    display: inline;
    line-height: 2;
  }

  > .score {
    font-size: .75em;
  }
}

.movie-overview {
  color: white;
  padding-top: 40px;
  padding-bottom: 40px;

  > .divider {
    border-top: 5px solid #fCD000;
  }

  > .summary {
    display: flex;
    padding-top: 20px;
    flex-direction: column;

    @media (min-width: 1200px) {
      flex-direction: row;
    }
  }
}

.movie-legend {
  display: flex;
  align-items: baseline;

  > .button {
    background: none;
    border: none;
    cursor: pointer;
  }

  > .divider {
    margin: 0 5px;
    font-size: 1em;
  }
  > .title {
    font-size: 1em;
    margin-left: 8px;
  }
}

.movie-content {
  > .image {
    align-self: center;
  }

  > .description {
    font-size: .75em;
  }
}
