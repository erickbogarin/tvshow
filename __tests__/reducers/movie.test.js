import { ActionTypes } from 'constants/index';
import movie from 'reducers/movie';

describe('Movie', () => {
  test('returns the initial state', () => {
    expect(movie(undefined, { type: {} })).toMatchSnapshot();
  });
});
