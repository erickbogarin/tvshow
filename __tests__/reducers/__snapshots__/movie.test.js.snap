exports[`Movie returns the initial state 1`] = `Object {}`;
